packaging/nnstreamer.spec

```diff
@@ -60,6 +60,15 @@
 %define		edgetpu_support 0
 %endif
 
+# tizen 6.0 (or less) backward-compatibility check
+%if ( 0%{?tizen_version_major} == 6 && 0%{?tizen_version_minor} < 5 ) || 0%{?tizen_version_major} < 6
+%define		grpc_support 0
+%define		tensorflow2_lite_support 0
+%ifarch x86_64
+%define		nnfw_support 0
```

Comment:
@again4you please check this line (nnfw in 6.0)

Reply:
1) Our Tizen customers request that they need `nnfw` on Tizen 6.0. So I am going to add them to Tizen 6.0 Platform image.
2) After https://github.com/nnstreamer/nnstreamer/pull/2949 is merged, `nnfw` and `nnstreamer-nnfw` supports all Architectures including `x86_64`. So this `if` statement is not necessary.

Reply:
I removed this line